person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] Kindly remove this `super` because it causes errors about wrong arguments. we use super in the `initialize` when the class you inherit from has the `initialize` method, also known as the constructor